astroid/nodes.py

```diff
@@ -24,6 +24,7 @@
 # Nodes not present in the builtin ast module:  DictUnpack, Unknown, and EvaluatedObject.
 
 from astroid.node_classes import (  # pylint: disable=redefined-builtin (Ellipsis)
+    CONST_CLS,
```

Comment:
I believe `CONST_CLS` is just used internally for astroid. It therefore can be accessed directly.

Reply:
You're right, I removed it.